tests/end_to_end/conftest.py

```diff
@@ -195,3 +196,48 @@ def relayers(worker_id):
     except ValueError:
         logger.info(f"Error while setting relayer index to {worker_id}")
     return
+
+
+# Uniswap fixtures
+
+TOTAL_SUPPLY = 10000 * 10**18
+
+
+@pytest_asyncio.fixture(scope="function")
```

Comment:
do you really want to deploy a new token for each test?

Reply:
Yes this is necessary for the UniswapV2 ERC20 tests